src/lib/types/Product.ts

```diff
@@ -78,10 +78,7 @@ export interface Product extends Timestamps, ProductTranslatableFields {
 		hideContentAfter: boolean;
 	};
 	hasVariations?: boolean;
-	variations?: {
-		name: string;
-		value: string;
-	}[];
+	variationPrices?: Record<string, Record<string, number>>;
```

Comment:
please switch it back, just add `price` optional in addition to `name` and `value`

Later we may want to add more things than just price, and we'll be able to add more optional props to `variations`

Reply:
Was too complicate to use ` variations?: {name: string; value: string; }[];` 
but let's try again